pom.xml

```diff
@@ -93,71 +186,6 @@
         <target.name>linux-aarch64</target.name>
       </properties>
     </profile>
-    <profile>
```

Comment:
This is profile is used for {{oss.sonatype.org}}, since we have add the parent {{commons-parent}}, it will contains release profile. https://commons.apache.org/commons-parent-pom.html#Release